### PR TITLE
chore: update the readme

### PR DESCRIPTION
**Pull request by violenil**
Finetuner Client README.md needs to be updated.

### CODE REVIEW

**Review by bwanglzu — CHANGES_REQUESTED** [review]
great work! left some comments, we'll need to fix the links in support section later

File: README.md [hunk 1]
```diff
@@ -1,4 +1,67 @@
-# Finetuner
+<p align="center">
+<img src="https://github.com/jina-ai/finetuner/blob/main/docs/_static/finetuner-logo-ani.svg?raw=true" alt="Finetuner logo: Finetuner helps you to create experiments with Finetuner in order to improve embeddings on search tasks. It accompanies Jina to deliver the last mile of performance-tuning for neural search applications." width="150px">
```
[the remark this answers is not in the document]
Comment (violenil) [reply README.md:2]: Not sure how to add this link in the alt text here.

**Review by jupyterjazz — APPROVED** [review]
Great work! :rocket:

**Review by jupyterjazz — COMMENTED** [review]
can you also change `resnet18` to `resnet50` in the docs?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,68 @@
...
+</p>
+
+<!-- start elevator-pitch -->
+
+Finetuning for neural search is not trivial, it requires expertise in ML and Information Retrieval which is a rare combination.
+Finetuner is a cloud service that makes finetuning simple and fast. Moving the process into the cloud, Finetuner handles all related complexity and infrastructure, making models performant and production ready.
+
+🎛 **Designed for finetuning**: a deep learning client for leveling up your pretrained models in domain-specific neural search applications.
+
+🍰 **Easy to use**: fine-tuning with Finetuner is a piece of cake! Easily create your fine-tuning experiments and runs in just a few lines of code by interacting with an API that runs the jobs for you.
```
Comment (CatStark) [inline README.md:22]:
```suggestion
🍰 **Easy to use**: Fine-tuning with Finetuner is a piece of cake! Easily create your fine-tuning experiments and runs in just a few lines of code by interacting with an API that runs the jobs for you.
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,68 @@
...
+<a href="https://pypi.org/project/finetuner/"><img src="https://img.shields.io/badge/Python-3.9%2B-blue alt="Python 3.9" title="Finetuner supports Python 3.9 and above"></a>
+<a href="https://slack.jina.ai"><img src="https://img.shields.io/badge/Slack-2.2k%2B-blueviolet?logo=slack&amp;logoColor=white"></a>
+</p>
+
+<!-- start elevator-pitch -->
+
+Finetuning for neural search is not trivial, it requires expertise in ML and Information Retrieval which is a rare combination.
+Finetuner is a cloud service that makes finetuning simple and fast. Moving the process into the cloud, Finetuner handles all related complexity and infrastructure, making models performant and production ready.
+
+🎛 **Designed for finetuning**: a deep learning client for leveling up your pretrained models in domain-specific neural search applications.
```
Comment (CatStark) [inline README.md:20]:
```suggestion
🎛 **Designed for finetuning**: A deep learning client for leveling up your pretrained models in domain-specific neural search applications.
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,113 @@
...
+
+<!-- start support-pitch -->
+## Support
+
+- Take a look at the [step by step](https://ft-docs-polish--jina-docs.netlify.app/2_step_by_step/) documentation for an overview of how Finetuner works.
+- Get started with our example use-cases in the [Finetuner in action](https://ft-docs-polish--jina-docs.netlify.app/3_finetuner_in_action/) section.
+- Use [Discussions](https://github.com/jina-ai/finetuner/discussions) to talk about your use cases, questions, and
+  support queries.
+- Join our [Slack community](https://slack.jina.ai) and chat with other Jina community members about ideas.
+- Join our [Engineering All Hands](https://youtube.com/playlist?list=PL3UBBWOUVhFYRUa_gpYYKBqEAkO4sxmne) meet-up to discuss your use case and learn Jina's new features.
```
Comment (CatStark) [inline README.md:102]:
```suggestion
- Join our [Engineering All Hands](https://youtube.com/playlist?list=PL3UBBWOUVhFYRUa_gpYYKBqEAkO4sxmne) meet-up to discuss your use case and learn Jina AI new features.
```

File: README.md [hunk 5]
```diff
@@ -1,4 +1,113 @@
...
+        loss='TripletMarginLoss',
+        callbacks=[BestModelCheckpoint(), EvaluationCallback(query_data='resnet-tll-eval-data')],
+        epochs=6,
+        learning_rate=0.001,
+    )
+
+print(run.status())
+print(run.logs())
+
+run.save_model('resnet-tll')
```
Comment (bwanglzu) [inline README.md:82]: can we make it minimal, do not include the arguments which can be fileed automatically, such as experiment name and run name

File: README.md [hunk 6]
```diff
@@ -1,4 +1,113 @@
...
+<!-- start elevator-pitch -->
+
+Fine-tuning deep neural networks (DNNs) significantly improves performance on domain specific neural search tasks.
+However, fine-tuning for neural search is not trivial, as it requires a combination of expertise in ML and Information Retrieval.
+Finetuner makes finetuning simple and fast by handling all related complexity and infrastructure in the cloud. With Finetuner, you can easily make models more perfomant and production ready.
+
+📈**Performance boost**: Finetuner significantly increases the performance of pretrained models on domain specific neural search applications.
+
+🔱 **Simple yet powerful**: Interacting with Finetuner is simple and seamless, and also supports rich features such as
+siamese/triplet loss, metric learning, self-supervised pretraining, layer pruning, weights freezing, dimensionality reduction, and much more.
```
Comment (bwanglzu) [inline README.md:24]: let's hide the self-supervised now, it's not added into the runner yet

File: README.md [hunk 7]
```diff
@@ -1,4 +1,113 @@
...
+
+☁ **Finetune in the cloud**: Finetuner runs your fine-tuning jobs on [Jina Cloud](https://github.com/jina-ai/jcloud). You never have to worry about provisioning (cloud) resources! Finetuner handles all related complexity and infrastructure.
```
Comment (bwanglzu) [inline README.md:26]: we need to remove the link of [Jina Cloud], jina cloud is not identical to JCloud. You can just mention:
>jobs on the cloud.

File: README.md [hunk 8]
```diff
@@ -1,4 +1,113 @@
...
+
+<!-- end elevator-pitch -->
+
+## What is the purpose of Finetuner?
+
+Finetuner enables performance gains on domain specific neural search tasks by allowing you to fine-tune models in the cloud. We have conducted experiments on three neural search tasks in different domains to illustrate these performance improvements.
```
Comment (bwanglzu) [inline README.md:32]: can we say experiments on varies tasks such as ..., not mention three tasks? we're actually experimenting on a lot of tasks. For me it's like we

File: README.md [hunk 9]
```diff
@@ -1,4 +1,113 @@
...
+<img src="https://github.com/jina-ai/finetuner/blob/docs-update-readme/docs/_static/finetuner-client-journey.svg?raw=true" title="Finetuner Client user journey.">
+
+
+## Install
+
+Requires Python 3.7+ installed on Linux/MacOS.
+
+```bash
+pip install -U finetuner-client
+```
```
Comment (bwanglzu) [inline README.md:53]: just a remark, we need to manage conda later

**Review by bwanglzu — COMMENTED** [review]
left some comments

File: README.md (hunk 9 above)
Comment (violenil) [reply README.md:53]: added this to our ToDo document.

**Comment by bwanglzu** [comment]
do not forget to add changelog

**Pull request closed without merging**